eve/workers/centos7_cloudinit/install_python3.sh

```diff
@@ -0,0 +1 @@
+yum install -y python34
```

Comment:
Guess it's not strictly required with the image we seem to be using, but this would require installing `epel-release` first on a bare image.